ios/RCTMGL-v10/RCTMGLMapView.swift

```diff
@@ -589,21 +595,38 @@ extension RCTMGLMapView: GestureManagerDelegate {
       }
     }
   }
-  
+    
+    private func resetWasGestureIndicators() {
+        if (self.wasGestureActive) {
+            self.wasGestureActive = false
+        }
+        if (self.wasAnimatingFromGesture) {
+            self.wasAnimatingFromGesture = false
+        }
+    }
+    
   public func gestureManager(_ gestureManager: GestureManager, didBegin gestureType: GestureType) {
     isGestureActive = true
   }
   
   public func gestureManager(_ gestureManager: GestureManager, didEnd gestureType: GestureType, willAnimate: Bool) {
-    isGestureActive = false
+     
     if willAnimate {
       isAnimatingFromGesture = true
+    } else {
+        wasGestureActive = true;
+        isGestureActive = false;
```

Comment:
```suggestion
     // will set isGestureActive to false from `didEndAnimatingFor`
   } else {
      isGestureActive = false;
   }
```

also remove `isAnimatingFromGesture = true`